### PR TITLE
P0 edit delete

### PR DESCRIPTION
**Pull request by dev-prabhat**
Implemented Deleted functionality and Archive Functionality 

- Check here https://github.com/dev-prabhat/note-it-down/pull/6/files

### CODE REVIEW

File: src/components/InputForm/InputForm.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
-import {useNote} from "../../context/Note-Context"
-import { BsFillPinFill,BsTrash } from "react-icons/bs";
+import {v4 as uuid} from "uuid"
+import {useNote,useArchive} from "../../context"
+
+import { BsFillPinFill } from "react-icons/bs";
 import { IoIosColorPalette , IoIosAddCircleOutline} from "react-icons/io";
 import { BiArchiveIn } from "react-icons/bi";
+
 import "./inputForm.css"
```
Comment (zoyatanweer) [inline src/components/InputForm/InputForm.js:9]: remove spacing

File: src/context/Archive-Context.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React,{createContext,useContext,useState} from "react"
+
+const ArchiveContext = createContext()
+
+const ArchiveProvider = ({children}) => {
+    const [archiveNotes, setArchiveNote] = useState([])
+
+    const addToArchive = (note) => setArchiveNote(prev => [...prev,note])
+
+
```
Comment (zoyatanweer) [inline src/context/Archive-Context.js:10]: remove spacing

**Review by zoyatanweer — APPROVED** [review]
- code is very clean and understandable
- good work 
- good to merge

File: src/components/InputForm/InputForm.js (hunk 1 above)
Comment (dev-prabhat) [reply src/components/InputForm/InputForm.js:9]: Space is given only for readability purposes

File: src/context/Archive-Context.js (hunk 2 above)
Comment (dev-prabhat) [reply src/context/Archive-Context.js:10]: Space is given only for readability purposes

**Review by srishtigautam21 — APPROVED** [review]
clean understandable code
good to merge

**Pull request merged**